tsl/test/shared/sql/dist_gapfill_pushdown.sql.in

```diff
@@ -0,0 +1,51 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+SET enable_partitionwise_aggregate = 'on';
+SET timescaledb.enable_remote_explain = true;
+
+-- Cases where gapfill is pushed down to data-nodes
+EXPLAIN (VERBOSE, COSTS OFF) SELECT time_bucket_gapfill('3 hours', time, '2017-01-01 06:00', '2017-01-01 18:00'),
```

Comment:
It would be nice to have a test case that outputs the actual data to see that the gapfill is producing the right output. It could also be compared to the output of a regular (non-distributed) hypertable.

Reply:
It is already done in dist_gapfill.sql when checking the diff for partitionwise_on and off. In case of partitionwise_on the second query in dist_gapfill_query.sql is pushing down gapfill.